Dockerfile

```diff
@@ -1,26 +1,25 @@
 FROM node:lts-alpine3.19 as builder
 
-COPY . ./
+COPY src ./src

```

Comment:
to benefit from the docker build cache you probably want to move the src after installing the dependencies:

```
COPY package.json .
COPY package-lock.json .
COPY tsconfig.json .

RUN npm install --legacy-peer-deps --omit=dev

COPY src ./src
COPY docs ./docs

RUN npm cache clean --force &&\
    npm run build:docs &&\
    npm run test
    ```